include/broker/zeek.hh

```diff
@@ -105,6 +202,22 @@ public:
     return get<std::string>(get<vector>(as_vector()[2])[0]);
   }
 
+  MetadataWrapper metadata() const {
+    const vector* md_vec_ptr = nullptr;
+    auto ev_vec_ptr = get_if<vector>(as_vector()[2]);
+    if (ev_vec_ptr && ev_vec_ptr->size() >= 3)
+      md_vec_ptr = get_if<vector>((*ev_vec_ptr)[2]);
+
+    return MetadataWrapper{md_vec_ptr};
```

Comment:
Having a variable that gets set somewhere along the way makes it harder to grasp the control flow. I think this would be more clear:

```suggestion
    if (auto ev_vec_ptr = get_if<vector>(as_vector()[2]);
        ev_vec_ptr && ev_vec_ptr->size() >= 3)
      return MetadataWrapper{get_if<vector>((*ev_vec_ptr)[2])};
    return MetadataWrapper{nullptr};
```